secant/Features/Home/Views/HomeView.swift

```diff
@@ -4,79 +4,59 @@ import ComposableArchitecture
 struct HomeView: View {
     let store: Store<HomeState, HomeAction>
 
-    var body: some View {
-        WithViewStore(store) { viewStore in
-            VStack {
-                Button(
-                    action: { viewStore.toggleShowingHistory() },
-                    label: { Text(viewStore.historyToggleString()) }
-                )
-                .primaryButtonStyle
-                .frame(height: 50)
+    var navigationRouteValues: [RouteValue] = HomeState.Route.allCases
+        .enumerated()
+        .filter { $0.1 != .history }
+        .map { RouteValue(id: $0.0, route: $0.1) }
 
-                Button(
-                    action: { viewStore.toggleSelectedTransaction() },
-                    label: { Text("Toggle Selected Transaction") }
-                )
-                .primaryButtonStyle
-                .frame(height: 50)
-
-                Button(
-                    action: { viewStore.send(.updateRoute(.recoveryPhraseDisplay)) },
-                    label: { Text("Show Recovery Phrase Demo") }
-                )
-                .primaryButtonStyle
-                .frame(height: 50)
+    var modalRoutes: [RouteValue] = HomeState.Route.allCases
+        .enumerated()
+        .filter { $0.1 == .history }
+        .map { RouteValue(id: $0.0, route: $0.1) }
 
-                Button(
-                    action: { viewStore.send(.updateRoute(.send)) },
-                    label: { Text("Go to Send") }
+    @ViewBuilder func view(for route: HomeState.Route) -> some View {
+        switch route {
+        case .history:
+            TransactionHistoryView(store: store.historyStore())
+        case .send:
+            SendView(
+                store: .init(
+                    initialState: .placeholder,
+                    reducer: SendReducer.default(
+                        whenDone: { HomeViewStore(store).send(.updateRoute(nil)) }
+                    ).debug(),
```

Comment:
Just curious, should we be leaving .debug in place? Not entirely sure if it matters or not, was just wondering if we have expectations there either way.

Reply:
Good question!

It's wrapped in `#if DEBUG`s within the method, and returns `self` i.e does nothing if we haven't  defined`DEBUG` https://github.com/pointfreeco/swift-composable-architecture/blob/bf9ab75f9dce30c510c5c7bf284225335f6a24a0/Sources/ComposableArchitecture/Debugging/ReducerDebugging.swift#L99-L144

So code wise it's fine, however stylistically it sets a bit of an odd precedent 🤔. Good call out, don't have a definitive position on it 😅. Erring on the side of "yes" since it's fine for release and super useful in development. But definitely feels a bit odd.

Reply:
Good question!

It's wrapped in `#if DEBUG`s within the method, and returns `self` i.e does nothing if we haven't  defined`DEBUG` https://github.com/pointfreeco/swift-composable-architecture/blob/bf9ab75f9dce30c510c5c7bf284225335f6a24a0/Sources/ComposableArchitecture/Debugging/ReducerDebugging.swift#L99-L144

So code wise it's fine, however stylistically it sets a bit of an odd precedent 🤔. Good call out, don't have a definitive position on it 😅. Erring on the side of "yes" since it's fine for release and super useful in development. But definitely feels a bit odd.

Reply:
[non-blocking] 
The `.debug` stuff should be unequivocally removed from builds shipped to testers at some point. the verbosity of the logs it's too much for a privacy-minded wallet like ours. Or at least we should make testers aware of this statements so they use it accordingly. they are really useful for debug. but they could disclose too much when used extensively.

Reply:
I wonder if we could implement a flavor of this ourselves, that serves us for debugging test builds that reproduce complex problems.